2021-05-17 15:56:25 [JobThread.run-130] <br>----------- dasher job execute start -----------<br>----------- Param:
2021-05-17 15:56:25 [BuildCommand.buildDataXParam-100] ------------------Command parameters:
2021-05-17 15:56:25 [ExecutorJobHandler.execute-57] ------------------DataX process id: 15392
2021-05-17 15:56:25 [ProcessCallbackThread.callbackLog-186] <br>----------- dasher job callback finish.
2021-05-17 15:56:25 [AnalysisStatistics.analysisStatisticsLog-53] 
2021-05-17 15:56:25 [AnalysisStatistics.analysisStatisticsLog-53] DataX (DATAX-OPENSOURCE-3.0), From Alibaba !
2021-05-17 15:56:25 [AnalysisStatistics.analysisStatisticsLog-53] Copyright (C) 2010-2017, Alibaba Group. All Rights Reserved.
2021-05-17 15:56:25 [AnalysisStatistics.analysisStatisticsLog-53] 
2021-05-17 15:56:25 [AnalysisStatistics.analysisStatisticsLog-53] Active code page: 65001
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 2021-05-17 15:56:26.563 [main] INFO  VMInfo - VMInfo# operatingSystem class => com.sun.management.internal.OperatingSystemImpl
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 2021-05-17 15:56:26.570 [main] INFO  Engine - the machine info  => 
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 	osInfo:	Oracle Corporation 11 11.0.9+7-LTS
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 	jvmInfo:	Windows 10 amd64 10.0
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 	cpu num:	8
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 	totalPhysicalMemory:	-0.00G
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 	freePhysicalMemory:	-0.00G
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 	maxFileDescriptorCount:	-1
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 	currentOpenFileDescriptorCount:	-1
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 	GC Names	[G1 Young Generation, G1 Old Generation]
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 	MEMORY_NAME                    | allocation_size                | init_size                      
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 	CodeHeap 'profiled nmethods'   | 117.19MB                       | 2.44MB                         
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 	G1 Old Gen                     | 1,024.00MB                     | 970.00MB                       
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 	G1 Survivor Space              | -0.00MB                        | 0.00MB                         
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 	CodeHeap 'non-profiled nmethods' | 117.19MB                       | 2.44MB                         
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 	Compressed Class Space         | 1,024.00MB                     | 0.00MB                         
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 	Metaspace                      | -0.00MB                        | 0.00MB                         
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 	G1 Eden Space                  | -0.00MB                        | 54.00MB                        
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 	CodeHeap 'non-nmethods'        | 5.63MB                         | 2.44MB                         
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 2021-05-17 15:56:26.590 [main] INFO  Engine - 
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] {
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 	"content":[
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 		{
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 			"reader":{
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 				"name":"mysqlreader",
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 				"parameter":{
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 					"column":[
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 						"`sid`",
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 						"`sname`",
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 						"`gender`",
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 						"`birth`",
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 						"`native_place`",
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 						"`politics_status`",
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 						"`phone`",
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 						"`major`"
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 					],
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 					"connection":[
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 						{
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 							"jdbcUrl":[
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 								"jdbc:mysql://127.0.0.1:3306/mysql_testdb?serverTimezone=UTC"
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 							],
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 							"table":[
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 								"se_test"
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 							]
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 						}
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 					],
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 					"password":"*****",
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 					"splitPk":"",
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 					"username":"root",
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 					"where":"native_place = '上海市'"
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 				}
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 			},
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 			"writer":{
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 				"name":"mysqlwriter",
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 				"parameter":{
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 					"column":[
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 						"`sid`",
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 						"`sname`",
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 						"`gender`",
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 						"`birth`",
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 						"`native_place`",
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 						"`politics_status`",
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 						"`phone`",
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 						"`major`"
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 					],
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 					"connection":[
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 						{
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 							"jdbcUrl":"jdbc:mysql://127.0.0.1:3306/mysql_testdb?serverTimezone=UTC",
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 							"table":[
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 								"se_test2"
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 							]
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 						}
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 					],
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 					"password":"*****",
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 					"username":"root"
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 				}
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 			}
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 		}
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 	],
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 	"setting":{
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 		"errorLimit":{
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 			"percentage":0.02,
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 			"record":0
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 		},
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 		"speed":{
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 			"byte":1048576,
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 			"channel":3
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 		}
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 	}
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] }
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 2021-05-17 15:56:26.610 [main] WARN  Engine - prioriy set to 0, because NumberFormatException, the value is: null
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 2021-05-17 15:56:26.612 [main] INFO  PerfTrace - PerfTrace traceId=job_-1, isEnable=false, priority=0
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 2021-05-17 15:56:26.612 [main] INFO  JobContainer - DataX jobContainer starts job.
2021-05-17 15:56:26 [AnalysisStatistics.analysisStatisticsLog-53] 2021-05-17 15:56:26.613 [main] INFO  JobContainer - Set jobId = 0
2021-05-17 15:56:27 [AnalysisStatistics.analysisStatisticsLog-53] 2021-05-17 15:56:27.377 [job-0] WARN  DBUtil - test connection of [jdbc:mysql://127.0.0.1:3306/mysql_testdb?serverTimezone=UTC] failed, for Code:[DBUtilErrorCode-10], Description:[连接数据库失败. 请检查您的 账号、密码、数据库名称、IP、Port或者向 DBA 寻求帮助(注意网络环境).].  -  具体错误信息为：com.mysql.jdbc.exceptions.jdbc4.MySQLNonTransientConnectionException: Could not create connection to database server..
2021-05-17 15:56:27 [AnalysisStatistics.analysisStatisticsLog-53] 2021-05-17 15:56:27.381 [job-0] ERROR RetryUtil - Exception when calling callable, 异常Msg:DataX无法连接对应的数据库，可能原因是：1) 配置的ip/port/database/jdbc错误，无法连接。2) 配置的username/password错误，鉴权失败。请和DBA确认该数据库的连接信息是否正确。
2021-05-17 15:56:27 [AnalysisStatistics.analysisStatisticsLog-53] java.lang.Exception: DataX无法连接对应的数据库，可能原因是：1) 配置的ip/port/database/jdbc错误，无法连接。2) 配置的username/password错误，鉴权失败。请和DBA确认该数据库的连接信息是否正确。
2021-05-17 15:56:27 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.plugin.rdbms.util.DBUtil$2.call(DBUtil.java:71) ~[plugin-rdbms-util-0.0.1-SNAPSHOT.jar:na]
2021-05-17 15:56:27 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.plugin.rdbms.util.DBUtil$2.call(DBUtil.java:51) ~[plugin-rdbms-util-0.0.1-SNAPSHOT.jar:na]
2021-05-17 15:56:27 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.common.util.RetryUtil$Retry.call(RetryUtil.java:164) ~[datax-common-0.0.1-SNAPSHOT.jar:na]
2021-05-17 15:56:27 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.common.util.RetryUtil$Retry.doRetry(RetryUtil.java:111) ~[datax-common-0.0.1-SNAPSHOT.jar:na]
2021-05-17 15:56:27 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.common.util.RetryUtil.executeWithRetry(RetryUtil.java:30) ~[datax-common-0.0.1-SNAPSHOT.jar:na]
2021-05-17 15:56:27 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.plugin.rdbms.util.DBUtil.chooseJdbcUrl(DBUtil.java:51) ~[plugin-rdbms-util-0.0.1-SNAPSHOT.jar:na]
2021-05-17 15:56:27 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.plugin.rdbms.reader.util.OriginalConfPretreatmentUtil.dealJdbcAndTable(OriginalConfPretreatmentUtil.java:92) ~[plugin-rdbms-util-0.0.1-SNAPSHOT.jar:na]
2021-05-17 15:56:27 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.plugin.rdbms.reader.util.OriginalConfPretreatmentUtil.simplifyConf(OriginalConfPretreatmentUtil.java:59) ~[plugin-rdbms-util-0.0.1-SNAPSHOT.jar:na]
2021-05-17 15:56:27 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.plugin.rdbms.reader.util.OriginalConfPretreatmentUtil.doPretreatment(OriginalConfPretreatmentUtil.java:33) ~[plugin-rdbms-util-0.0.1-SNAPSHOT.jar:na]
2021-05-17 15:56:27 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.plugin.rdbms.reader.CommonRdbmsReader$Job.init(CommonRdbmsReader.java:55) ~[plugin-rdbms-util-0.0.1-SNAPSHOT.jar:na]
2021-05-17 15:56:27 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.plugin.reader.mysqlreader.MysqlReader$Job.init(MysqlReader.java:37) ~[mysqlreader-0.0.1-SNAPSHOT.jar:na]
2021-05-17 15:56:27 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.core.job.JobContainer.initJobReader(JobContainer.java:673) ~[datax-core-0.0.1-SNAPSHOT.jar:na]
2021-05-17 15:56:27 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.core.job.JobContainer.init(JobContainer.java:303) ~[datax-core-0.0.1-SNAPSHOT.jar:na]
2021-05-17 15:56:27 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.core.job.JobContainer.start(JobContainer.java:113) ~[datax-core-0.0.1-SNAPSHOT.jar:na]
2021-05-17 15:56:27 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.core.Engine.start(Engine.java:92) ~[datax-core-0.0.1-SNAPSHOT.jar:na]
2021-05-17 15:56:27 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.core.Engine.entry(Engine.java:171) ~[datax-core-0.0.1-SNAPSHOT.jar:na]
2021-05-17 15:56:27 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.core.Engine.main(Engine.java:204) ~[datax-core-0.0.1-SNAPSHOT.jar:na]
2021-05-17 15:56:28 [AnalysisStatistics.analysisStatisticsLog-53] 2021-05-17 15:56:28.406 [job-0] ERROR RetryUtil - Exception when calling callable, 即将尝试执行第1次重试.本次重试计划等待[1000]ms,实际等待[1014]ms, 异常Msg:[DataX无法连接对应的数据库，可能原因是：1) 配置的ip/port/database/jdbc错误，无法连接。2) 配置的username/password错误，鉴权失败。请和DBA确认该数据库的连接信息是否正确。]
2021-05-17 15:56:28 [AnalysisStatistics.analysisStatisticsLog-53] 2021-05-17 15:56:28.409 [job-0] WARN  DBUtil - test connection of [jdbc:mysql://127.0.0.1:3306/mysql_testdb?serverTimezone=UTC] failed, for Code:[DBUtilErrorCode-10], Description:[连接数据库失败. 请检查您的 账号、密码、数据库名称、IP、Port或者向 DBA 寻求帮助(注意网络环境).].  -  具体错误信息为：com.mysql.jdbc.exceptions.jdbc4.MySQLNonTransientConnectionException: Could not create connection to database server..
2021-05-17 15:56:30 [AnalysisStatistics.analysisStatisticsLog-53] 2021-05-17 15:56:30.422 [job-0] ERROR RetryUtil - Exception when calling callable, 即将尝试执行第2次重试.本次重试计划等待[2000]ms,实际等待[2012]ms, 异常Msg:[DataX无法连接对应的数据库，可能原因是：1) 配置的ip/port/database/jdbc错误，无法连接。2) 配置的username/password错误，鉴权失败。请和DBA确认该数据库的连接信息是否正确。]
2021-05-17 15:56:30 [AnalysisStatistics.analysisStatisticsLog-53] 2021-05-17 15:56:30.425 [job-0] WARN  DBUtil - test connection of [jdbc:mysql://127.0.0.1:3306/mysql_testdb?serverTimezone=UTC] failed, for Code:[DBUtilErrorCode-10], Description:[连接数据库失败. 请检查您的 账号、密码、数据库名称、IP、Port或者向 DBA 寻求帮助(注意网络环境).].  -  具体错误信息为：com.mysql.jdbc.exceptions.jdbc4.MySQLNonTransientConnectionException: Could not create connection to database server..
2021-05-17 15:56:34 [AnalysisStatistics.analysisStatisticsLog-53] 2021-05-17 15:56:34.427 [job-0] ERROR RetryUtil - Exception when calling callable, 即将尝试执行第3次重试.本次重试计划等待[4000]ms,实际等待[4001]ms, 异常Msg:[DataX无法连接对应的数据库，可能原因是：1) 配置的ip/port/database/jdbc错误，无法连接。2) 配置的username/password错误，鉴权失败。请和DBA确认该数据库的连接信息是否正确。]
2021-05-17 15:56:34 [AnalysisStatistics.analysisStatisticsLog-53] 2021-05-17 15:56:34.430 [job-0] WARN  DBUtil - test connection of [jdbc:mysql://127.0.0.1:3306/mysql_testdb?serverTimezone=UTC] failed, for Code:[DBUtilErrorCode-10], Description:[连接数据库失败. 请检查您的 账号、密码、数据库名称、IP、Port或者向 DBA 寻求帮助(注意网络环境).].  -  具体错误信息为：com.mysql.jdbc.exceptions.jdbc4.MySQLNonTransientConnectionException: Could not create connection to database server..
2021-05-17 15:56:42 [AnalysisStatistics.analysisStatisticsLog-53] 2021-05-17 15:56:42.439 [job-0] ERROR RetryUtil - Exception when calling callable, 即将尝试执行第4次重试.本次重试计划等待[8000]ms,实际等待[8009]ms, 异常Msg:[DataX无法连接对应的数据库，可能原因是：1) 配置的ip/port/database/jdbc错误，无法连接。2) 配置的username/password错误，鉴权失败。请和DBA确认该数据库的连接信息是否正确。]
2021-05-17 15:56:42 [AnalysisStatistics.analysisStatisticsLog-53] 2021-05-17 15:56:42.441 [job-0] WARN  DBUtil - test connection of [jdbc:mysql://127.0.0.1:3306/mysql_testdb?serverTimezone=UTC] failed, for Code:[DBUtilErrorCode-10], Description:[连接数据库失败. 请检查您的 账号、密码、数据库名称、IP、Port或者向 DBA 寻求帮助(注意网络环境).].  -  具体错误信息为：com.mysql.jdbc.exceptions.jdbc4.MySQLNonTransientConnectionException: Could not create connection to database server..
2021-05-17 15:56:58 [AnalysisStatistics.analysisStatisticsLog-53] 2021-05-17 15:56:58.448 [job-0] ERROR RetryUtil - Exception when calling callable, 即将尝试执行第5次重试.本次重试计划等待[16000]ms,实际等待[16005]ms, 异常Msg:[DataX无法连接对应的数据库，可能原因是：1) 配置的ip/port/database/jdbc错误，无法连接。2) 配置的username/password错误，鉴权失败。请和DBA确认该数据库的连接信息是否正确。]
2021-05-17 15:56:58 [AnalysisStatistics.analysisStatisticsLog-53] 2021-05-17 15:56:58.450 [job-0] WARN  DBUtil - test connection of [jdbc:mysql://127.0.0.1:3306/mysql_testdb?serverTimezone=UTC] failed, for Code:[DBUtilErrorCode-10], Description:[连接数据库失败. 请检查您的 账号、密码、数据库名称、IP、Port或者向 DBA 寻求帮助(注意网络环境).].  -  具体错误信息为：com.mysql.jdbc.exceptions.jdbc4.MySQLNonTransientConnectionException: Could not create connection to database server..
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 2021-05-17 15:57:30.454 [job-0] ERROR RetryUtil - Exception when calling callable, 即将尝试执行第6次重试.本次重试计划等待[32000]ms,实际等待[32003]ms, 异常Msg:[DataX无法连接对应的数据库，可能原因是：1) 配置的ip/port/database/jdbc错误，无法连接。2) 配置的username/password错误，鉴权失败。请和DBA确认该数据库的连接信息是否正确。]
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 2021-05-17 15:57:30.458 [job-0] WARN  DBUtil - test connection of [jdbc:mysql://127.0.0.1:3306/mysql_testdb?serverTimezone=UTC] failed, for Code:[DBUtilErrorCode-10], Description:[连接数据库失败. 请检查您的 账号、密码、数据库名称、IP、Port或者向 DBA 寻求帮助(注意网络环境).].  -  具体错误信息为：com.mysql.jdbc.exceptions.jdbc4.MySQLNonTransientConnectionException: Could not create connection to database server..
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 2021-05-17 15:57:30.581 [job-0] ERROR JobContainer - Exception when job run
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] com.alibaba.datax.common.exception.DataXException: Code:[DBUtilErrorCode-10], Description:[连接数据库失败. 请检查您的 账号、密码、数据库名称、IP、Port或者向 DBA 寻求帮助(注意网络环境).].  - 数据库连接失败. 因为根据您配置的连接信息,无法从:jdbc:mysql://127.0.0.1:3306/mysql_testdb?serverTimezone=UTC 中找到可连接的jdbcUrl. 请检查您的配置并作出修改. - java.lang.Exception: DataX无法连接对应的数据库，可能原因是：1) 配置的ip/port/database/jdbc错误，无法连接。2) 配置的username/password错误，鉴权失败。请和DBA确认该数据库的连接信息是否正确。
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.plugin.rdbms.util.DBUtil$2.call(DBUtil.java:71)
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.plugin.rdbms.util.DBUtil$2.call(DBUtil.java:51)
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.common.util.RetryUtil$Retry.call(RetryUtil.java:164)
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.common.util.RetryUtil$Retry.doRetry(RetryUtil.java:111)
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.common.util.RetryUtil.executeWithRetry(RetryUtil.java:30)
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.plugin.rdbms.util.DBUtil.chooseJdbcUrl(DBUtil.java:51)
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.plugin.rdbms.reader.util.OriginalConfPretreatmentUtil.dealJdbcAndTable(OriginalConfPretreatmentUtil.java:92)
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.plugin.rdbms.reader.util.OriginalConfPretreatmentUtil.simplifyConf(OriginalConfPretreatmentUtil.java:59)
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.plugin.rdbms.reader.util.OriginalConfPretreatmentUtil.doPretreatment(OriginalConfPretreatmentUtil.java:33)
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.plugin.rdbms.reader.CommonRdbmsReader$Job.init(CommonRdbmsReader.java:55)
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.plugin.reader.mysqlreader.MysqlReader$Job.init(MysqlReader.java:37)
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.core.job.JobContainer.initJobReader(JobContainer.java:673)
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.core.job.JobContainer.init(JobContainer.java:303)
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.core.job.JobContainer.start(JobContainer.java:113)
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.core.Engine.start(Engine.java:92)
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.core.Engine.entry(Engine.java:171)
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.core.Engine.main(Engine.java:204)
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.common.exception.DataXException.asDataXException(DataXException.java:33) ~[datax-common-0.0.1-SNAPSHOT.jar:na]
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.plugin.rdbms.util.DBUtil.chooseJdbcUrl(DBUtil.java:77) ~[plugin-rdbms-util-0.0.1-SNAPSHOT.jar:na]
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.plugin.rdbms.reader.util.OriginalConfPretreatmentUtil.dealJdbcAndTable(OriginalConfPretreatmentUtil.java:92) ~[plugin-rdbms-util-0.0.1-SNAPSHOT.jar:na]
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.plugin.rdbms.reader.util.OriginalConfPretreatmentUtil.simplifyConf(OriginalConfPretreatmentUtil.java:59) ~[plugin-rdbms-util-0.0.1-SNAPSHOT.jar:na]
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.plugin.rdbms.reader.util.OriginalConfPretreatmentUtil.doPretreatment(OriginalConfPretreatmentUtil.java:33) ~[plugin-rdbms-util-0.0.1-SNAPSHOT.jar:na]
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.plugin.rdbms.reader.CommonRdbmsReader$Job.init(CommonRdbmsReader.java:55) ~[plugin-rdbms-util-0.0.1-SNAPSHOT.jar:na]
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.plugin.reader.mysqlreader.MysqlReader$Job.init(MysqlReader.java:37) ~[mysqlreader-0.0.1-SNAPSHOT.jar:na]
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.core.job.JobContainer.initJobReader(JobContainer.java:673) ~[datax-core-0.0.1-SNAPSHOT.jar:na]
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.core.job.JobContainer.init(JobContainer.java:303) ~[datax-core-0.0.1-SNAPSHOT.jar:na]
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.core.job.JobContainer.start(JobContainer.java:113) ~[datax-core-0.0.1-SNAPSHOT.jar:na]
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.core.Engine.start(Engine.java:92) ~[datax-core-0.0.1-SNAPSHOT.jar:na]
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.core.Engine.entry(Engine.java:171) ~[datax-core-0.0.1-SNAPSHOT.jar:na]
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.core.Engine.main(Engine.java:204) ~[datax-core-0.0.1-SNAPSHOT.jar:na]
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] Caused by: java.lang.Exception: DataX无法连接对应的数据库，可能原因是：1) 配置的ip/port/database/jdbc错误，无法连接。2) 配置的username/password错误，鉴权失败。请和DBA确认该数据库的连接信息是否正确。
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.plugin.rdbms.util.DBUtil$2.call(DBUtil.java:71) ~[plugin-rdbms-util-0.0.1-SNAPSHOT.jar:na]
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.plugin.rdbms.util.DBUtil$2.call(DBUtil.java:51) ~[plugin-rdbms-util-0.0.1-SNAPSHOT.jar:na]
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.common.util.RetryUtil$Retry.call(RetryUtil.java:164) ~[datax-common-0.0.1-SNAPSHOT.jar:na]
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.common.util.RetryUtil$Retry.doRetry(RetryUtil.java:111) ~[datax-common-0.0.1-SNAPSHOT.jar:na]
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.common.util.RetryUtil.executeWithRetry(RetryUtil.java:30) ~[datax-common-0.0.1-SNAPSHOT.jar:na]
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.plugin.rdbms.util.DBUtil.chooseJdbcUrl(DBUtil.java:51) ~[plugin-rdbms-util-0.0.1-SNAPSHOT.jar:na]
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	... 11 common frames omitted
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 2021-05-17 15:57:30.587 [job-0] INFO  StandAloneJobContainerCommunicator - Total 0 records, 0 bytes | Speed 0B/s, 0 records/s | Error 0 records, 0 bytes |  All Task WaitWriterTime 0.000s |  All Task WaitReaderTime 0.000s | Percentage 0.00%
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 2021-05-17 15:57:30.589 [job-0] ERROR Engine - 
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 经DataX智能分析,该任务最可能的错误原因是:
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] com.alibaba.datax.common.exception.DataXException: Code:[DBUtilErrorCode-10], Description:[连接数据库失败. 请检查您的 账号、密码、数据库名称、IP、Port或者向 DBA 寻求帮助(注意网络环境).].  - 数据库连接失败. 因为根据您配置的连接信息,无法从:jdbc:mysql://127.0.0.1:3306/mysql_testdb?serverTimezone=UTC 中找到可连接的jdbcUrl. 请检查您的配置并作出修改. - java.lang.Exception: DataX无法连接对应的数据库，可能原因是：1) 配置的ip/port/database/jdbc错误，无法连接。2) 配置的username/password错误，鉴权失败。请和DBA确认该数据库的连接信息是否正确。
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.plugin.rdbms.util.DBUtil$2.call(DBUtil.java:71)
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.plugin.rdbms.util.DBUtil$2.call(DBUtil.java:51)
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.common.util.RetryUtil$Retry.call(RetryUtil.java:164)
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.common.util.RetryUtil$Retry.doRetry(RetryUtil.java:111)
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.common.util.RetryUtil.executeWithRetry(RetryUtil.java:30)
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.plugin.rdbms.util.DBUtil.chooseJdbcUrl(DBUtil.java:51)
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.plugin.rdbms.reader.util.OriginalConfPretreatmentUtil.dealJdbcAndTable(OriginalConfPretreatmentUtil.java:92)
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.plugin.rdbms.reader.util.OriginalConfPretreatmentUtil.simplifyConf(OriginalConfPretreatmentUtil.java:59)
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.plugin.rdbms.reader.util.OriginalConfPretreatmentUtil.doPretreatment(OriginalConfPretreatmentUtil.java:33)
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.plugin.rdbms.reader.CommonRdbmsReader$Job.init(CommonRdbmsReader.java:55)
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.plugin.reader.mysqlreader.MysqlReader$Job.init(MysqlReader.java:37)
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.core.job.JobContainer.initJobReader(JobContainer.java:673)
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.core.job.JobContainer.init(JobContainer.java:303)
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.core.job.JobContainer.start(JobContainer.java:113)
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.core.Engine.start(Engine.java:92)
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.core.Engine.entry(Engine.java:171)
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.core.Engine.main(Engine.java:204)
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.common.exception.DataXException.asDataXException(DataXException.java:33)
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.plugin.rdbms.util.DBUtil.chooseJdbcUrl(DBUtil.java:77)
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.plugin.rdbms.reader.util.OriginalConfPretreatmentUtil.dealJdbcAndTable(OriginalConfPretreatmentUtil.java:92)
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.plugin.rdbms.reader.util.OriginalConfPretreatmentUtil.simplifyConf(OriginalConfPretreatmentUtil.java:59)
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.plugin.rdbms.reader.util.OriginalConfPretreatmentUtil.doPretreatment(OriginalConfPretreatmentUtil.java:33)
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.plugin.rdbms.reader.CommonRdbmsReader$Job.init(CommonRdbmsReader.java:55)
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.plugin.reader.mysqlreader.MysqlReader$Job.init(MysqlReader.java:37)
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.core.job.JobContainer.initJobReader(JobContainer.java:673)
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.core.job.JobContainer.init(JobContainer.java:303)
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.core.job.JobContainer.start(JobContainer.java:113)
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.core.Engine.start(Engine.java:92)
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.core.Engine.entry(Engine.java:171)
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.core.Engine.main(Engine.java:204)
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] Caused by: java.lang.Exception: DataX无法连接对应的数据库，可能原因是：1) 配置的ip/port/database/jdbc错误，无法连接。2) 配置的username/password错误，鉴权失败。请和DBA确认该数据库的连接信息是否正确。
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.plugin.rdbms.util.DBUtil$2.call(DBUtil.java:71)
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.plugin.rdbms.util.DBUtil$2.call(DBUtil.java:51)
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.common.util.RetryUtil$Retry.call(RetryUtil.java:164)
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.common.util.RetryUtil$Retry.doRetry(RetryUtil.java:111)
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.common.util.RetryUtil.executeWithRetry(RetryUtil.java:30)
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	at com.alibaba.datax.plugin.rdbms.util.DBUtil.chooseJdbcUrl(DBUtil.java:51)
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 	... 11 more
2021-05-17 15:57:30 [AnalysisStatistics.analysisStatisticsLog-53] 
2021-05-17 15:57:30 [JobThread.run-165] <br>----------- dasher job execute end(finish) -----------<br>----------- ReturnT:ReturnT [code=500, msg=command exit value(1) is failed, content=null]
2021-05-17 15:57:30 [TriggerCallbackThread.callbackLog-186] <br>----------- dasher job callback finish.
2021-05-17 15:58:17 [JobThread.run-130] <br>----------- dasher job execute start -----------<br>----------- Param:null
2021-05-17 15:58:17 [ProcessUtil.killProcessByPid-76] Active code page: 65001
2021-05-17 15:58:17 [JobThread.run-165] <br>----------- dasher job execute end(finish) -----------<br>----------- ReturnT:ReturnT [code=200, msg=null, content=null]
